### PR TITLE
Trading Bot Docs Updates

### PR DESCRIPTION
**Pull request by vraptor75011**
Trading bot update to specify stop/start running tasks after dependencies edit

### CODE REVIEW

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Bot/trading-bot.json [hunk 1]
```diff
@@ -125,7 +125,7 @@
                 },
                 {
                     "language": "DE",
-                    "text": "Trading Mines erleichtern die Entwicklung von Trading Bots, indem sie einen Rahmen für die Erstellung unterteilter Definitionen für die Prozesse und Produkte der Bots bieten, ähnlich wie Data Mines es für Sensoren und Indikatoren tun.",
+                    "text": "Trading Mines erleichtern die Entwicklung von Trading Bots, indem sie einen Rahmen für die Erstellung unterteilter Definitionen für die Prozesse und Produkte der Bots bieten, ��hnlich wie Data Mines es für Sensoren und Indikatoren tun.",
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Bot/trading-bot.json:128]: The ä doesn't seem to be detected properly, you can try to fix it and do a commit again or if that's too cumbersome for you I can merge it and afterwards we'll adjust it. :+1:

Comment (vraptor75011) [reply Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Bot/trading-bot.json:128]: Hi, is the translation to German Automatic ? I didn t write it

**Review by vraptor75011 — COMMENTED** [review]
Tested it locally and it worked

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Bot/trading-bot.json (hunk 1 above)
Comment (vraptor75011) [reply Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Bot/trading-bot.json]: Done

**Review by gitforkgit — APPROVED** [review]

**Comment by gitforkgit** [comment]
merged

**Pull request merged**